src/cmdline/UriHandler.cpp

```diff
@@ -25,28 +25,28 @@ extern "C"
     #include "../openrct2.h"
 }
 
-static exitcode_t HandleUri(const utf8 * uri);
-
 exitcode_t CommandLine::HandleCommandUri(CommandLineArgEnumerator * enumerator)
 {
     const utf8 * uri;
     if (enumerator->TryPopString(&uri))
     {
         if (String::StartsWith(uri, "openrct2://"))
         {
-            const utf8 * uriCommand = uri + 11;
-            return HandleUri(uriCommand);
+            const utf8 * uriCommand = uri;
+            return (exitcode_t) HandleUri(uriCommand);
         }
     }
 
     Console::Error::WriteLine("Invalid URI");
     return EXITCODE_FAIL;
 }
 
-static exitcode_t HandleUri(const utf8 * uri)
+extern "C"
+int HandleUri(const utf8 * uri)
 {
+	assert(String::StartsWith(uri, "openrct2://"));
     utf8 * * args;
-    size_t numArgs = String::Split(&args, uri, '/');
+    size_t numArgs = String::Split(&args, uri + 11, '/');
```

Comment:
@LRFLEW for gcc, it does get optimised out, and hopefully it still does when using `String::SizeOf`. But its not particularly important.